Interfaces/IGame.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using CribblyBackend.Models;
+
+namespace CribblyBackend.Interfaces
+{
+    public interface IGame
```

Comment:
Semantically, I view interfaces as a contract of operation: anything that implements an interface can do X, Y, Z. I wonder if it makes more sense here to implement the base as a class to be inherited from instead of an interface?

Reply:
The main reason I did this is so that we can have a list of all games, regardless of type. In the last Cribbly, I ended up having to make a list for each game type. Unless you think there is a better way, or if having separate lists for each type will be best?

Reply:
I think a division has `Game`s associated with it, and a bracket/tournament has `Game`s associated with it. It seems like the data shape can actually be the same though. There are two teams, two scores, a winner, etc. The score differential doesn't really matter for tournament games, but we can just ignore it. I think it's simplest to use one model until we find a reason not to 🤷‍♂️